### PR TITLE
Qol 8082 clikable card enhacements

### PR DESCRIPTION
**Pull request by asifamingov**

### CODE REVIEW

File: src/assets/_project/_blocks/components/forms/qg-recaptcha.js [hunk 1]
```diff
@@ -66,66 +64,49 @@ import keys from '../../data/qg-google-keys';
...
     /**
-     * From SWE4 onwards footer feedback is AJAX based
+     * from swe4 onwards footer feedback is ajax based
+     * footerFeedbackRecaptcha function handles recaptcha on ajax based form.
+     * This function creates a submit event
+     * Call submission handler api
+     * Get and display success message
      * @return {undefined}
      **/
     footerFeedbackRecaptcha: function() {
```
Comment (duttonw) [inline src/assets/_project/_blocks/components/forms/qg-recaptcha.js:74]: This is badly named looking at the comments.  can you renamed to 
footerFeedbackSubmitWithRecaptchaCheck

File: .gitignore [hunk 2]
```diff
@@ -54,3 +54,4 @@ release
 package-lock.json
 web-template-release/
 template-pages/sample/*
+/src/docs/examples/
```
Comment (duttonw) [inline .gitignore:57]: why ignore examples?

**Review by duttonw — COMMENTED** [review]
Is this on the test site yet, need to verify with our multi page captcha system  i.e. https://qld-gov-au.github.io/qg-web-template/captcha.html

File: .gitignore (hunk 2 above)
Comment (asifamingov) [reply .gitignore:57]: these are examples copied from qld.gov.au just for debugging issues, exact pages so suppose some sensitive information is deleted on qld.gov.au but is available in SWE repo won't be good.

File: src/assets/_project/_blocks/components/forms/qg-recaptcha.js (hunk 1 above)
Comment (asifamingov) [reply src/assets/_project/_blocks/components/forms/qg-recaptcha.js]: updated

**Comment by asifamingov** [comment]
> Is this on the test site yet, need to verify with our multi page captcha system i.e. https://qld-gov-au.github.io/qg-web-template/captcha.html

I verified multiple recaptcha on this page https://oss-uat.clients.squiz.net/contact-us . I removed that extra script which Mitch added and on this page https://oss-uat.clients.squiz.net/contact-us/responsive-maintenance-request which contains V2 and V3

File: src/assets/_project/_blocks/components/forms/qg-recaptcha.js [hunk 3]
```diff
@@ -66,66 +64,49 @@ import keys from '../../data/qg-google-keys';
...
-            }
-          });
+    footerFeedbackSubmitWithRecaptchaCheck: function() {
+      var self = this;
+      grecaptcha.ready(() => {
+        $('#qg-page-feedback-form').submit(function (event) {
+          var targetFormSubmit = $(this);
+          event.preventDefault();
+          if ($('#qg-page-feedback-form li.invalid').length <= 0) {
+            var greptcha = self.config.$feedbackForm.find('input[name="g-recaptcha-response"]');
```
Comment (ThrawnCA) [inline src/assets/_project/_blocks/components/forms/qg-recaptcha.js:81]: This is a somewhat confusing variable name, especially with `grecaptcha` already existing.

**Review by ThrawnCA — APPROVED** [review]

File: src/assets/_project/_blocks/components/forms/qg-recaptcha.js (hunk 3 above)
Comment (asifamingov) [reply src/assets/_project/_blocks/components/forms/qg-recaptcha.js]: updated in the latest commit

**Pull request merged**